gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/portal_pages/PortalPagesResource.java

```diff
@@ -47,4 +54,16 @@ public PortalPageResponse getPortalHomepage() {
 
         return PortalPagesMapper.INSTANCE.map(homepage);
     }
+
+    @PUT
+    @Produces("application/json")
+    @Consumes("application/json")
+    @Path("/{pageId}")
+    @Permissions({ @Permission(value = RolePermission.API_DOCUMENTATION, acls = { RolePermissionAction.UPDATE }) })
```

Comment:
```suggestion
    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_DOCUMENTATION, acls = { RolePermissionAction.UPDATE }) })
```

Shouldn't it be ENVIRONMENT level permission here ? 🤔